packages/fxa-settings/src/components/RecoveryKeySetupHint/index.test.tsx

```diff
@@ -110,7 +110,7 @@ describe('RecoveryKeySetupHint', () => {
     const textInput = screen.getByRole('textbox', {
       name: 'Enter a hint (optional)',
     });
-    await waitFor(() => user.type(textInput, hintValueTooLong));
+    await user.type(textInput, hintValueTooLong);
```

Comment:
The majority of the test updates here is because we have to use `user.type` since it correctly does the input validation cycle.